src/bxbot/Services/Pairing/PairingService.cs

```diff
@@ -15,6 +16,42 @@ public PairingService(IRestConnector restConnector)
             this.restConnector = restConnector;
         }
 
+        public async Task<IEnumerable<SelectOption>> GetCurrenciesAsync()
+        {
+            var selectOptions = new List<SelectOption>();
+            var url = "https://bx.in.th/api/pairing/";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code not to use hardcoded absolute paths or URIs. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1075)